.github/workflows/go.yml

```diff
@@ -13,11 +13,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.13
+    - name: Set up Go 1.14
       uses: actions/setup-go@v1
       with:
-        go-version: 1.13
+        go-version: 1.14
```

Comment:
before merging this we should probably have a standup to inform everyone that they need to update their go, or can we keep it at 1.13 for now?

Reply:
don't we all use 1.14?

Reply:
Sorry, yes, I thought it was an upgrade of what we are running.